changelog.md

```diff
@@ -108,4 +108,6 @@
   * Support magic v2 batch/message format (message headers).
   * Use rebar3 as primary build tool for travis-ci, erlang.mk is still supported.
   * Support `SCRAM-SHA-256` and `SCRAM-SHA-512` SASL authentication mechanisms.
+* 3.?.?
```

Comment:
Update version number.

Reply:
not planing to release it now.
will update when we decide to release.